### PR TITLE
Enable CRD scale

### PR DESCRIPTION
**Pull request by jtaleric**
Adding CRD Scale as a workload, also adding it to the control-plane workflow.

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-payload-control-plane-6nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-payload-control-plane-6nodes

**Comment by jtaleric** [comment]
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/53310/rehearse-53310-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-payload-control-plane-6nodes/1803044881946382336#1:build-log.txt%3A249

need to see where this is getting injected... I took out `churn-duration`

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-payload-control-plane-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-payload-control-plane-6nodes

**Review by chentex — APPROVED** [review]
LGTM

File: ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ chain:
   as: openshift-qe-control-plane
   steps:
   - chain: openshift-qe-cluster-density-v2
+  - chain: openshift-qe-crd-scale
```
Comment (vishnuchalla) [inline ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml:6]: I am not sure if we want to keep it as a part of this chain or separate.

**Comment by vishnuchalla** [comment]
looks good to me in overall.

File: ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml (hunk 1 above)
Comment (jtaleric) [reply ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml:6]: IMHO it makes the most sense to have in our control-plane work flow to encourage better utilization and $ spend

Comment (jtaleric) [reply ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml:6]: make sense @vishnuchalla ?

Comment (vishnuchalla) [reply ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml:6]: Yes.

**Comment by jtaleric** [comment]
image
Bumping the iterations to 2000 to see impact

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-payload-control-plane-6nodes

**Comment by jtaleric** [comment]
image

Moving to 2k for our payload testing.

**Pull request merged**